cdc/processor/sinkmanager/table_sink_wrapper.go

```diff
@@ -105,6 +105,14 @@ func newTableSinkWrapper(
 		startTs:    startTs,
 		targetTs:   targetTs,
 	}
+	res.checkpointTs.Store(startTs)
+	for {
```

Comment:
I think there is no need to use a for-loop spin to compare and swap. Because before we create it there is no way to update receivedSorterResolvedTs.